internal/graphql/otel_graphql_engine_detailed_test.go

```diff
@@ -480,6 +480,10 @@ func (s *spanWrapper) TracerProvider() trace.TracerProvider {
 	return s.span.TracerProvider()
 }
 
+func (s *spanWrapper) AddLink(link trace.Link) {
+	s.span.AddLink(link)
+}
+
```

Comment:
New method added by the library that must be implemented to satisfy the interface.